lib/instrumentations.js

```diff
@@ -10,39 +10,40 @@ const InstrumentationDescriptor = require('./instrumentation-descriptor')
 // Return a new copy of this array every time we're called
 module.exports = function instrumentations() {
   return {
```

Comment:
I sorted the lines in these objects. I find it a lot easier to read names in alphabetical order, and to figure out where to add a new item when the list is in alphabetical order.